_pages/faq.md

```diff
@@ -5,31 +5,64 @@ permalink: /faq/
 
-Where there is additional information about the status of indicator data collection and reporting, this will be displayed at the top of the indicator page.
+- [What does the reporting status mean?](#what-does-the-reporting-status-mean)
+
+- [Can other countries copy this website?](#can-other-countries-copy-this-website)
+
+- [How accessible is this website?](#how-accessible-is-this-website)
+
+- [What browsers can I use to view this website?](#what-browsers-can-i-use-to-view-this-website)
+
+- [What do each of the three tiers mean?](#what-do-each-of-the-three-tiers-mean)
 
 ## How often will new data be added to this site?
 We will add data as soon as it has been sourced and checked. This includes new data as well as updates to data we have already published. We will continue to collect data from existing sources where possible, in cooperation with topic experts.
 
+## How do I suggest new or different data sources?
+If you have feedback on the data sources we have used or have suggestions for new data sources then please email us at <a href="mailto:{{site.email_contacts.questions}}">{{site.email_contacts.questions}}</a>.
+
 ## What are you doing to fill data gaps?
 For some indicators, although we may have data at the national level, our existing data sources don’t always allow us to disaggregate by all main groups.
 
 This means we need to look for new data sources, link existing ones, or model data.
 
+## What does the reporting status mean?
+We have used three different types of reporting status for an indicator, which are also colour coded:
+
+- **Reported (black):** As a minimum the headline national data for this indicator is available on this website but the data might not be fully disaggregated yet. We are continuing to source additional disaggregations.
+
+- **Statistics in progress (grey):** We have found a suitable source of data for this indicator or relevant proxy at national level. We are currently quality assuring the data and preparing it for publication.
+
+- **Exploring data sources (white):** We are still looking for a suitable data source for this indicator.
+
+- **Not applicable (striped):** This indicator is not applicable in the national context.
+
+Where there is additional information about the status of indicator data collection and reporting, this will be displayed at the top of the indicator page.
+
+## Can other countries copy this website?
+Yes. This site is built and hosted using 100% free software and services, so other countries and localities can freely and easily reuse our code. Technical guidance on implementing this platform is available [here](https://open-sdg.readthedocs.io). Countries and localities exploring how to report their own SDGs data can also get support from the [Center for Open Data Enterprise (CODE)](http://www.opendataenterprise.org/) through their [SDG National Reporting Initiative](https://www.sdgreporting.org/).
+
 ## How accessible is this website?
```

Comment:
@AnnCorp @brockfanning wondering if we need to change the answer to this question so it says something more like 

Open SDG aims to be as accessible and usable as possible for every user and constantly works towards meeting AA level of  [Web Content Accessibility Guidelines (WCAG 2.0)](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag-20). However...

I'm just thinking if a country takes a copy of Open SDG and doesn't update the FAQ page but makes customisation, the answer as it is may be misleading